### DISCUSSION TITLE
Wow - Love v5 beta

### DISCUSSION THREAD

**Discussion by AnonJohn** [mbnuqw/sidebery#837] [General]
Hi mnuqw -Just dropping by to say the the beta version 5 is amazing. I had been grinding away to make small UI tweaks in v4, slowly making it the way I like and getting frustrated on some lingering details.

I installed version 5 and promptly threw away the customization - no longer needed - and was delighted to find that the linger things were also all fixed. Wonderful design choices. And the styles editor is clearer too. Cheers!

**Comment by zakkhoyt** [comment]
Hey, I wanted to drop a comment here too. I agree with Anon. I was recently combing through the settings and GitHub and noticed the 5.x releases. My plugin version was 4.x. I was pleased to find that I could install a beta channel and promptly did. Really digging the improvements and bug fixes. 

Most folks at my work are switching to [Arc Browser](https://arc.net/), but it's FF + Sidebery for me.

**Comment by Qjo1** [comment]
Same experience I had. v5 is fantastic. Wonder why it wasn't pushed to AMO yet.
